RXPromise Libraries/Source/RXPromise.mm

```diff
@@ -146,15 +146,13 @@ @implementation RXPromise {
     id                  _result;
     RXPromise_State     _state;
     
-    std::list<std::pair<id,promise_progressHandler_t>> *_progressHandlers;
-    OSSpinLock __volatile                               _progressSpinLock;
-    std::list<RXPromise* __weak>                       *_children;
+    std::list<std::pair<id,promise_progressHandler_t>> _progressHandlers;
+    std::list<RXPromise* __weak>                       _children;
```

Comment:
ivar `_children` can possibly be omitted. Please read my comments, and verify my assumptions.